rabbit-core/src/main/java/org/ohdsi/utilities/files/QuickAndDirtyXlsxReader.java

```diff
@@ -521,10 +515,18 @@ private void processWorkBook(InputStream inputStream) throws NumberFormatExcepti
 				Sheet sheet = rIdToSheet.get(rId);
 				sheet.setName(name);
 				sheet.order = Integer.parseInt(order);
+				nameToSheet.put(name, sheet);
 			}
 		}
 	}
 
+	public Sheet getByName(String sheetName) {
+		if (nameToSheet.containsKey(sheetName)) {
+			return nameToSheet.get(sheetName);
+		}
+		return null;

```

Comment:
This implementation is identical to just returning `nameToSheet.get(sheetName)`. It will return `null` if the key is not present.